ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java

```diff
@@ -48,7 +46,7 @@ public void XSSOnPhoneNumberFieldTest() throws Exception {
         registrationEditSectionPage.clearPhoneNumber();
         registrationEditSectionPage.enterPhoneNumber("<script>alert(0)</script>");
         registrationEditSectionPage.clickOnConfirmEdit();
-        assertThat(registrationEditSectionPage.getValidationErrors(), is(not(empty())));
+        assertThat(registrationEditSectionPage.getValidationErrors(), is(empty()));
```

Comment:
Any reason for this replacement ?

Reply:
cc @k-joseph

Reply:
@kdaud the test fails with a null pointer exception with both chrome and firefox but when i change, it passes on chrome with both headless true and false and fails on firefox.

Reply:
> but when i change

Have you had a look on what the `assertion` is achieving ?

Reply:
Thanks @Daud. Great clue.